spec/classes/motd_spec.rb

```diff
@@ -213,6 +213,7 @@
       let(:params) do
         {
           content: 'Hello!',
+	  windows_motd_title: 'This is the title.',
```

Comment:
Looks like the indentation is off here. Hard tabs should not be used, only spaces.